language/pt_br/common.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+*
+* About us extension for the phpBB Forum Software package.
+* [Brazilian Portuguese [pt_br]]
+* Brazilian Portuguese translation by null2 and update by eunaumtenhoid (c) 2017 [ver 2.1.2] (https://github.com/phpBBTraducoes)
```

Comment:
Please remove the version and the copyright here. We have one in line 7 and the version-number needs to be increased in every release. That is just more work. Git will take care of versioning.